module/applications/api/application-v2-mixin.mjs

```diff
@@ -57,6 +57,28 @@ export default function ApplicationV2Mixin(Base) {
 
     /* -------------------------------------------- */
 
+    /** @inheritDoc */
+    _onFirstRender(context, options) {
+      super._onFirstRender(context, options);
+      const containers = {};
+      for ( const [part, config] of Object.entries(this.constructor.PARTS) ) {
+        if ( !config.containerClasses ) continue;
+        const element = this.element.querySelector(`[data-application-part="${part}"]`);
+        if ( !element ) continue;
+        const className = config.containerClasses.join(" ");
```

Comment:
Using the classes for this seems fragile and also a conflation of concerns. If we want to do something like this I think it would be better to have a separate `container` identifier and `containerClasses`.